utils/node_local_test.py

```diff
@@ -2662,6 +2662,10 @@ def test_cont_copy(self):
         src_dir = tempfile.TemporaryDirectory(prefix='copy_src_',)
         with open(join(src_dir.name, 'file'), 'w') as ofd:
             ofd.write('hello')
+        cwd = os.getcwd()
+        os.chdir(src_dir.name)
+        os.symlink('file', 'file_s')
+        os.chdir(cwd)
```

Comment:
This will probably work well enough to get results, however these tests are run multi-threaded with other tests running concurrently but you can't land it like this.  As a suggestion (untested):
```suggestion
        os.symlink('file', join(src_dir.name, 'file_s'))
```

I assume this code would also work with broken symlinks as it should never deference them?

Reply:
thx! Just updated the test code. Yes. os.symlink() works with broken symbolic links too.